lib/console.js

```diff
@@ -83,7 +83,10 @@ Console.prototype.trace = function trace() {
 
 Console.prototype.assert = function(expression) {
   if (!expression) {
-    var arr = Array.prototype.slice.call(arguments, 1);
+    var al = arguments.length - 1;
+    var arr = new Array(al > 0 ? al : 0);
+    while (al-- > 0) arr[al] = arguments[al + 1];
+
     require('assert').ok(false, util.format.apply(this, arr));
```

Comment:
There's another obvious performance optimization here: cache the result of `require('assert')`. :-)